Applications/spire/source/time_and_sales/time_and_sales_window.cpp

```diff
@@ -244,6 +246,15 @@ void time_and_sales_window::set_properties(
   QFontMetrics metrics(m_properties.m_font);
   auto row_height = metrics.height() + scale_height(2);
   m_table->verticalHeader()->setDefaultSectionSize(row_height);
+  if(m_properties.m_font.pointSize() >= 11) {
+    m_header_font.setPointSizeF(m_properties.m_font.pointSize() * 0.8);
+  } else {
+    m_header_font.setPointSize(9);
+  }
+  m_table->horizontalHeader()->setFont(m_header_font);
+  QFontMetrics header_metrics(m_header_font);
+  m_table->horizontalHeader()->setFixedHeight(header_metrics.height() * 1.8);
+  repaint();
```

Comment:
What is the reason for using repaint instead of update?
What is the reason for forcing an update right away?